packages/mdc-drawer/standard/mdc-drawer-standard.scss

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/rtl/mixins";
+@import "@material/typography/mixins";
+@import "@material/typography/variables";
+@import "@material/theme/variables";
+@import "@material/theme/mixins";
+@import "../variables";
+@import "../mixins";
+
+$mdc-drawer-width: 256px;
```

Comment:
Why are these here rather than in `variables`?

Reply:
I didn't use that variable anywhere else, but perhaps a user would. I'll move it.